### ISSUE TITLE
Error when building solution involving /opencv2/videoio.hpp

### ISSUE THREAD

**Issue by lealsergio** [ms-iot/petdoor#7]
I found the pet door project on hackster and I am trying to replicate it on a Raspberry Pi. I have included all the built binaries from OpenCv according to the instructions however I am receiving an error that says the following:
![image](https://user-images.githubusercontent.com/36055840/35707383-20d8a8c6-076f-11e8-9118-5f6140858573.png)

it seems to include the opencv2/core.hpp but somehow not the videoio.hpp

I am stuck trying to find the correct path. Any help is appreciated

**Comment by dcrck** [comment]
Hi @lealsergio, can you validate that you did the following:
1. Built OpenCV for ARM
2. Included the videoio module DLL
3. Included the videoio Include Directory in your Additional Include Directories (%OCV2015_ROOT%\modules\videoio\include)
4. Included the videoio library in your Additional Dependencies (opencv_videoio300.lib)

I'm asking because the OpenCV sample referenced on Hackster does not include videoio since it's not needed there.

**Comment by lealsergio** [comment]
Hey @derekameer , thanks for the tips! 
I included the dll and the path. I got the app to build successfully but when I deployed the app, I got the following error:
![image](https://user-images.githubusercontent.com/36055840/35782250-e5766356-09ba-11e8-832d-1d590310446c.png)
I also see this message at the bottom:
![image](https://user-images.githubusercontent.com/36055840/35782429-6b63bdf4-09bd-11e8-9547-ab1c0b880599.png)

Could it be that there is a problem with the circuitry attached to the pi? I am looking into this myself but any advice is greatly appreciated
